less/index.less

```diff
@@ -25,35 +107,318 @@ time, mark, audio, video {
 	vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
+
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
 	display: block;
 }
+
 body {
 	line-height: 1;
 }
-ol, ul {
+
+ol,
+ul {
 	list-style: none;
 }
-blockquote, q {
+
+blockquote,
+q {
 	quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
+
+blockquote:after,
+blockquote:before,
+q:after,
+q:before {
 	content: '';
 	content: none;
 }
+
 table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
-
 /* Set every element's box-sizing to border-box */
+
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+.wrapper {
+	margin: 0 auto;
+	width: 1080px;
+	max-width: 100%;
+	height: 100%;
+	@media @phone {
+		width: 500px;
+		max-width: 100%;
+		height: 100%;
+	}
+
+	body,
+	html {
+		margin: 0;
+		/*font-size: 62.5%;*/
+	}
+}
+
+.nav {
+	border-left: 1px solid black;
+	border-right: 1px solid black;
+	margin-top: 20px;
+	margin-bottom: 75px;
+	width: 100%;
+	height: auto;
+	@media @phone {
+		margin: 0 auto 100px;
+		.center;
```

Comment:
During lecture yesterday Cam mentioned how you might  want to include the empty parenthesis in these mixins so that it's clear to whoever is viewing your code.